src/components/PassphraseDialog/PassphraseModal/PassphraseModal.scss

```diff
@@ -0,0 +1,54 @@
+@import "constants/style";
+$color-form-background--dark: #292f3b;
+
+.password-modal {
+  position: absolute;
+  backdrop-filter: blur(4px);
+  width: 100%;
+  height: 100%;
+}
+
+.password-modal__header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.password-modal__container {
+  padding: $spacing--lg;
+  background: #ffffff;
+  border-radius: 0.5em;
+  box-shadow: 0 10px 20px rgba(black, 0.2);
+  left: 50%;
+  max-width: 90%;
+  position: absolute;
+  top: 50%;
+  transform: translate(-50%, -50%);
```

Comment:
so I'm assuming you centered the container here. Can you make the the parent container flex instead and use `align-items` and `justify-content` instead, or is there any specific reason to do it this way?